src/core/metrics/app-metrics.ts

```diff
@@ -83,4 +117,55 @@ export class AppMetrics {
       }
     });
   }
+
+  watchList = {
+    // _done_userResourceAtom: null, // should be done before init
+    _done_currentEventResource: null,
+    _done_layersGlobalResource: null,
+    _done_analyticsResource: null,
+    // _done_areaLayersDetailsResourceAtom: null, // can be disable in url
+    _done_layersInAreaAndEventLayerResource: null,
+    // _done_eventListResource: null,
+    [EVENT_MAP_IDLE]: null,
+  };
+
+  watch(name: string) {
+    if (this.watchList[name] === null) {
+      const timing = performance.now();
+      this.watchList[name] = timing;
+      if (every(this.watchList, Boolean)) {
+        // last event should be EVENT_MAP_IDLE
+        if (name !== EVENT_MAP_IDLE) {
+          this.watchList[EVENT_MAP_IDLE] = null;
+          return;
+        }
+
+        setTimeout(() => {
+          this.report('ready', timing);
+          const eventReadyEvent = new Event('event_ready_for_screenshot');
+          window.dispatchEvent(eventReadyEvent);
+        }, 299);
+        return;
+      }
+      this.report(name, timing);
+    }
+  }
+
+  report(name: string, timing: number) {
+    // Endpoint: POST /active/api/rum/metrics
+
+    const payload = {
+      ...this.reportTemplate,
+      name,
+      value: timing,
+    };
+    console.warn('metrics', payload);
+    fetch(APP_METRICS_ENDPOINT, {
+      method: 'POST',
+      headers: { 'Content-Type': 'application/json' },
+      body: JSON.stringify([payload]),
+    }).catch((error) => {
+      console.error('sent metrics error:', error, payload);
+    });
```

Comment:
why not use standart `apiClientInstance` since it's based on `config.apiGateway` as well?